tmt/utils.py

```diff
@@ -786,8 +786,8 @@ def verdict(decision, comment=None, good='pass', bad='fail', problem='warn'):
     """
     Return verdict in green or red based on the decision
 
-    0 or False ... good (green)
-    1 or True .... bad (red)
+    0 or False .... bad (red)
```

Comment:
@lukaszachy I have a separate PR for this here: https://github.com/psss/tmt/pull/613

Reply:
@thrix You are changing implementation. This one just fixes misleading docstring. 
If this would block merge I will remove this change.

Reply:
On the second though - I will remove docstring change.

Reply:
yeah, ty, I am changing to comply to what @psss wanted to see, thanks